contribution-guide.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: ballerina-inner-page
+title: The Ballerina Contribution Guide
+permalink: /contribution-guide/
+---
+
+# The Ballerina Contribution Guide
+
+Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is licensed under the Apache license and is nurtured by all the contributors of the Ballerina community.
+
+We appreciate your help!
+
+- [Get started](#get-started)
+- [Set up the Prerequisites](#set-up-the-prerequisites)
+- [Set up the developer environment](#set-up-the-developer-environment)
+    - [Set up IntelliJ IDEA](#set-up-intellij-idea)
+    - [Set up Eclipse](#set-up-eclipse)
+- [Obtain the source code](#obtain-the-source-code)
+- [Build the project](#build-the-project)
+- [Contribute to Ballerina grammar](#contribute-to-ballerina-grammar)
+- [Submit your contribution](#submit-your-contribution)
+- [Accept the Contributor License Agreement](#accept-the-contributor-license-agreement)
+- [Propose changes to Ballerina](#propose-changes-to-ballerina)
+
+## Get started
+
+- Join the conversations at:
```

Comment:
What is the reason for removing the following line from this section? IMO, Since we are actively using github for discussions related to issues and new proposals, we should add it also.l

"GitHub: file issues, comment on other issues, send your pull requests."

Reply:
Added the link as follows:

 - [GitHub][(https://github.com/ballerina-platform/ballerina-lang/issues): to file issues, comment on other issues, send your pull requests.